gdnative-async/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "gdnative-async"
+authors = ["The godot-rust developers"]
+description = "Runtime async support for godot-rust."
+documentation = "https://docs.rs/crate/gdnative-async"
+repository = "https://github.com/godot-rust/godot-rust"
+homepage = "https://godot-rust.github.io/"
+version = "0.9.3"
+license = "MIT"
+workspace = ".."
+edition = "2018"
+
+[features]
+
+[dependencies]
+gdnative-derive = { path = "../gdnative-derive", version = "=0.9.3" }
+gdnative-core = { path = "../gdnative-core", version = "=0.9.3" }
+gdnative-bindings = { path = "../gdnative-bindings", version = "=0.9.3" }
+futures-task = "0.3.13"
```

Comment:
I didn't find any runtimes that supported futures-task traits explicitly. Is it available for tokio / async-executor?

Reply:
[`futures-executor`](https://crates.io/crates/futures-executor) supports the API, naturally. [`async_executors`](https://crates.io/crates/async_executors) claims to implement the trait for tokio and async-std executors.

In any case, locking users into either tokio or async-std is a no from me, and futures-rs is as close to std as we can get for now, being under `rust-lang` with a real chance of becoming std in the future. If there is a set of executor traits that are better adopted, please let me know.

Reply:
What a great find! Thank you for the reference.